lib/errors/errors_tests.c

```diff
@@ -153,7 +153,7 @@ fatal_errors_trigger(enum error_case_e type)
 #endif
 #ifdef CONFIG_ORB_LIB_WATCHDOG
     case FATAL_WATCHDOG:
-        (void)watchdog_init();
+        (void)watchdog_init(NULL);
```

Comment:
Can you check that this triggers a watchdog reset? 🤔 

I am not sure it did. And it's very likely that this won't fix it.
Make sure to pass a callback that returns `false`

Reply:
will check this